tests/test_utils.py

```diff
@@ -17,6 +17,21 @@
     _PyperplanHeuristicWrapper
 
 
+def test_aabb_volume():
+    """Tests for get_aabb_volume."""
+    lo = np.array([1.0, 1.5, -1.0])
+    hi = np.array([2.0, 2.5, 0.0])
+    assert utils.get_aabb_volume(lo, hi) == 1.0
+
+
+def test_aabb_closest_point():
+    """Tests for get_closest_point_on_aabb."""
+    xyz = [1.5, 3.0, -2.5]
+    lo = np.array([1.0, 1.5, -1.0])
+    hi = np.array([2.0, 2.5, 0.0])
+    assert utils.get_closest_point_on_aabb(xyz, lo, hi) == [1.5, 2.5, -1.0]
```

Comment:
won't this test only hit one code path of your implementation above? add tests to cover all the code paths, and let's split the utils.py changes into a separate PR so that we can verify coverage!

Reply:
It actually hits all paths (because the x coord is between lo and hi, the y coord is too big, and the z coord is too small).
I'll add the assertion check though!